pkg/miniogw/config.go

```diff
@@ -21,6 +22,15 @@ import (
 	"storj.io/storj/pkg/transport"
 )
 
+var (
+	maxInlineSize int
```

Comment:
Yeah I was also thinking it could be added to the existing ClientConfig or part of a new SegmentConfig, but I think @jtolds said it would be better as a flag for now.